code/modules/mob/living/carbon/human/life/handle_regular_hud_updates.dm

```diff
@@ -64,6 +64,8 @@
 			overlay_fullscreen("eye_blurry", /atom/movable/screen/fullscreen/impaired, 5)
 		else if((disabilities & NEARSIGHTED) && !HAS_TRAIT(src, TRAIT_NEARSIGHTED_EQUIPMENT))
 			overlay_fullscreen("eye_blurry", /atom/movable/screen/fullscreen/impaired, 2)
+		else if((disabilities & PHOTOSENSITIVITY) && !HAS_TRAIT(src, TRAIT_PHOTOSENSITIVITY_EQUIPMENT))
```

Comment:
As stated above, this seems highly unnecessary and a misuse of traits when there's a roughly good enough alternative that is already in use.